Certain-Chickens/Bangazon-Certain-Chickens
# Kevin

Ticket: [https://github.com/Certain-Chickens/Bangazon-Certain-Chickens/issues/9
](https://github.com/Certain-Chickens/Bangazon-Certain-Chickens/issues/9
)
Description: New TrainingProgram controller created for manipulating the database table TrainingProgram.

Steps to test:

git fetch --all
git checkout kevin
code .
dotnet run
enter your running server url into Postman
EX= http://LocalHost:5000/api/TrainingProgram
Change setting to POST, select BODY, select RAW, and select JSON
Enter new entry in JSON format without primary key and any foreign keys, click SEND
HERE is an example you can copy and paste
```
{
    "startDate": "2/2/18",
    "endDate": "2/3/18",
    "maxAttendees": 25
}
```
You will see the entry in your SQLite browser
Switch setting to GET and click SEND
You will see all of the current entries in JSON format
Add the ComputerId number you want to get as a single view to the URL (../TrainingProgram/{TrainingProgramId})
Click SEND
Switch setting to PUT and insure the ID of the entry you want to edit is at the end of the URL
Enter the ENTIRE JSON object into the entry field including the TrainingProgramId and its value then edit what you want to change
Click SEND and check the entry in your SQLite browser for changes.
Switch setting to DELETE and insure the ID of the entry you want to delete is at the end of the URL
There should be no JSON and click SEND
